app/src/main/java/com/khaithu/a3ctech_android_final_project/view/MovieDetailView.kt

```diff
@@ -0,0 +1,60 @@
+package com.khaithu.a3ctech_android_final_project.view
+
+import android.annotation.SuppressLint
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import androidx.core.view.isVisible
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+import com.khaithu.a3ctech_android_final_project.R
+import com.khaithu.a3ctech_android_final_project.model.Genre
+import com.khaithu.a3ctech_android_final_project.model.MovieDetail
+import com.khaithu.a3ctech_android_final_project.presenter.GetMoviePresenter
+import com.khaithu.a3ctech_android_final_project.view.interfaceView.IMovieDetailView
+import jp.wasabeef.glide.transformations.BlurTransformation
+import kotlinx.android.synthetic.main.activity_movie_detail_view.*
+
+class MovieDetailView : AppCompatActivity(), IMovieDetailView {
+
+    private val getMoviePresenter: GetMoviePresenter = GetMoviePresenter(this)
+    private val backdropBaseUrl = "https://www.themoviedb.org/t/p/w533_and_h300_bestv2/"
+    private val posterBaseUrl = "https://www.themoviedb.org/t/p/w600_and_h900_bestv2/"
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_movie_detail_view)
+
+        val intent: Intent = intent
+        val id: Int = intent.getIntExtra("id", 0)
+        getMoviePresenter.getMovieDetail(id)
+    }
+
+    @SuppressLint("SetTextI18n")
+    override fun onMovieDetailReceived(movieDetail: MovieDetail) {
+        Glide.with(this).load(posterBaseUrl + movieDetail.posterPath)
+            .into(posterMovie)
+
+        Glide.with(this).load(backdropBaseUrl + movieDetail.backdropPath)
+            .apply(RequestOptions.bitmapTransform(BlurTransformation(5)))
+            .into(posterBackground)
+
+        titleDetailMovie.text = "${movieDetail.title} (${movieDetail.releaseDate.substring(0, 4)})"
```

Comment:
You should write new method for MovieDetail instead of `movieDetail.releaseDate.substring(0, 4)`